src/trinerdi/sc.sh

```diff
@@ -1,12 +1,13 @@
 alias t="basename \$(realpath .)" # Assumption: prints an alphanumeric string
```

Comment:
I think it would be cleaner to move this inside `e()` and change to
```bash
t=$(pwd|xargs basename)
```
This should be functionally equivalent, right?

Reply:
Thanks for the tip. The `alias` hack is a relict from the time when there were more functions that used it, I guess.